acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/BitbucketScmWorkflowMultiBranchJob.java

```diff
@@ -32,10 +32,7 @@ public void reIndex() {
         try {
             super.reIndex();
         } catch (NoSuchElementException e) {
-            // JENKINS-63044
```

Comment:
Could you keep this comment? This is _guaranteed_ to bite us again in the future

Reply:
I removed it because there is a more informative one above on line 28.Perhaps you didn't see it since it's hidden in the diff view, but if you have seen it and still want the comment added back in, I can do so.